pkg/auth/cli.go

```diff
@@ -9,22 +9,24 @@ import (
 // LoginOptions represents common flags in login
 // caller should define bool or optionalBool fields for flags --get-login and --tls-verify
 type LoginOptions struct {
-	AuthFile      string
```

Comment:
Please split this, using comments, into CLI flags (`AuthFile` and the like) that are end-user specified, and Go options (`AcceptUnspecifiedRegistry`, `Stdin`) that are caller-specified:
```go
// CLI flags managed by the FlagSet returned by GetLoginFlags
    AuthFile string
    …
// CLI flags the caller must define manually (using …)
	GetLoginSet   bool // documentation of the flag name&description would be nice
// Options the caller can set
    AcceptUnspecifiedRegistry bool // individual member documentation would be nice
    Stdin                     io.Reader // …
    …
```
or something like that.

Applies similarly to the logout code.